<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareServiceWindow</class>
 <widget class="QFrame" name="HardwareServiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>395</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="wContainer">
     <widget class="QWidget" name="wMainPage"/>
     <widget class="QWidget" name="wEditorPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
